2-debugging-frames/README.md

```diff
@@ -0,0 +1,41 @@
+# E2: Debugging Frames
+
+Concert designers have dreamed up a revolutionary tippy apple juice concept. The animated mockup is wowing clients across verticals! Most of the ad is built. There are just a couple bugs left to be ironed out. That is your job!
+
+If you do not finish that is just fine. We are more interested in your approach. Keep track of steps you take to debug so we can have a conversation about tricky problems!
+
```

Comment:
Just so we are clear about how long we expect them to spend on this:

"Take some time to explore the code and debug -- please spend **no longer than an hour** with the project. If you do not finish that is just fine! We are more interested in your approach to problem solving and debugging. **Keep track of the steps you take to debug** so we can have a conversation about tricky problems!"